app/controllers/account/companies_controller.rb

```diff
@@ -1,5 +1,8 @@
+require 'bing_api'
```

Comment:
what is this?
there is no need to require gems if this is one

Reply:
it is my file at /lib directory
i hope it have to plugged like that

Reply:
it doesn't work like this
check [this](http://guides.rubyonrails.org/autoloading_and_reloading_constants.html#autoload-paths) out

Reply:
config.autoload_paths << "#{Rails.root}/lib" at the config/application.rb does not works(

Reply:
it does work, just don't forget to follow naming convention and paths